tests/ci/build_status_check.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+import json
+import time
+from urllib.request import urlopen
+
+def update_url_data(response):
+  data = json.loads(response.read().decode())
+  indices=[]
+  for n in range(data["total_count"]):
+    if re.search("Build", data["jobs"][n]["name"]):
+      indices.append(n)
+
+  if len(indices) == 0:
+    sys.exit(1)
```

Comment:
`raise SystemExit('Some explanation here')` does the same thing, should try to raise an `Exception` instead of using sys.exit()

Reply:
Done.